CorosHeader header
int32 message_code
int32 robot_id

